client/src/app/+accounts/account-video-channels/account-video-channels.component.html

```diff
@@ -19,6 +19,7 @@
         <div class="no-results my-5" i18n *ngIf="getVideosOf(videoChannel).length === 0">This channel doesn't have any videos.</div>
 
         <my-video-miniature
+          class="pr-2"
```

Comment:
Do we want right padding on mobile?

Reply:
It doesn't have any impact on mobile from what I see.